radicle-cli/src/commands/remote/add.rs

```diff
@@ -0,0 +1,41 @@
+use radicle::node::tracking::store::Config;
+use radicle::{git::Url, node::TRACKING_DB_FILE, prelude::Id, Profile};
+use radicle_crypto::PublicKey;
+
+use crate::git::rad_add_remote;
+use crate::{git, terminal as term};
+
+pub fn run(
+    repository: &git::Repository,
+    profile: &Profile,
+    pubkey: &PublicKey,
+    name: Option<String>,
+    id: Id,
+) -> anyhow::Result<()> {
+    // looks like that the clippy suggestion is wrong, and
+    // because name is not a function!
+    #[allow(clippy::redundant_closure)]
+    let Some(name) = name.map_or(get_remote_name(profile, pubkey)?, |name| Some(name)) else {
```

Comment:
I think what it's suggesting is this:
```suggestion
    let Some(name) = name.map_or(get_remote_name(profile, pubkey)?, Some) else {
```

Reply:
yes I just fixed this in the last push :) looks strange but works